services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/TransactWriteItemsEnhancedRequest.java

```diff
@@ -67,7 +67,45 @@ public static Builder builder() {
     }
 
     /**
-     * Gets the ClientRequestToken of this enhanced request.
+     * <p>
+     * Providing a <code>ClientRequestToken</code> makes the call to <code>TransactWriteItems</code> idempotent, meaning
+     * that multiple identical calls have the same effect as one single call.
+     * </p>
+     * <p>
+     * Although multiple identical calls using the same client request token produce the same result on the server (no
+     * side effects), the responses to the calls might not be the same. If the <code>ReturnConsumedCapacity&gt;</code>
```

Comment:
We need to remove the stuff about ReturnConsumedCapacity as the enhanced client doesn't support it right now.